en/docs/learn/api-security/oauth2/grant-types/authorization-code-grant.md

```diff
@@ -63,52 +69,67 @@ The steps below show how access tokens are generated for the authorization code
 !!! note
     **Before you begin**
 
-    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/5.9.0/develop/deploying-the-sample-app/) .
+    The following instructions use the sample playground webapp. For instructions on how to set up the sample webapp, see [Setting up the Sample Webapp](https://is.docs.wso2.com/en/5.9.0/develop/deploying-the-sample-app/).
 
 
-1.  Log in to the API Developer Portal and create a new application.
+1.  Open a terminal window and add the following entry to the `/etc/hosts` file of your machine to configure the hostname.
+    ```
+    127.0.0.1   wso2is.local
+    127.0.0.1   localhost.com
+    ```
+        
+    !!!info "Why is this step needed?"
+        Some browsers do not allow you to create cookies for a naked hostname, such as `localhost`. Cookies are required when working with SSO. Therefore, to ensure that the SSO capabilities work as expected in this tutorial, you need to configure the `/etc/host` file as explained in this step.
+            
+        The `/etc/host` file is a read-only file. Therefore, you won't be able to edit it by opening the file via a text editor. Instead, edit the file using the terminal commands. For example, use the following command if you are working on a Mac/Linux environment.
```

Comment:
I think the file name should be changed as `/etc/hosts`?

Reply:
fixed with 34f4e03